x-pack/plugins/infra/public/pages/link_to/redirect_to_logs.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { InjectedIntl, injectI18n } from '@kbn/i18n/react';
+import compose from 'lodash/fp/compose';
+import React from 'react';
+import { Redirect, RouteComponentProps } from 'react-router-dom';
+
+import { LoadingPage } from '../../components/loading_page';
+import { replaceLogFilterInQueryString } from '../../containers/logs/with_log_filter';
+import { replaceLogPositionInQueryString } from '../../containers/logs/with_log_position';
+import { WithSource } from '../../containers/with_source';
+import { getFilterFromLocation, getTimeFromLocation } from './query_params';
+
+type RedirectToLogsType = RouteComponentProps<{}>;
+
+interface RedirectToLogsProps extends RedirectToLogsType {
+  intl: InjectedIntl;
+}
+
+export const RedirectToLogs = injectI18n(({ location, intl }: RedirectToLogsProps) => (
+  <WithSource>
+    {({ configuration, isLoading }) => {
+      if (isLoading) {
+        return (
+          <LoadingPage
+            message={intl.formatMessage({
+              id: 'xpack.infra.redirectToNodeLogs.loadingNodeLogsMessage',
```

Comment:
This id is already taken with a different message, which seems to cause the CI failure.

```suggestion
              id: 'xpack.infra.redirectToLogs.loadingLogsMessage',
```